src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -257,6 +281,14 @@ public void setParticipation(Participation participation) {
         this.participation = participation;
     }
 
+    public Set<SubmissionVersion> getVersions() {
+        return versions;
+    }
+
+    public void setVersions(Set<SubmissionVersion> submissionVersions) {
+        this.versions = submissionVersions;
+    }
+
```

Comment:
Is there a reason for adding getters/setters? If this is on purpose, it should probably be added to the PR description/title

Reply:
I added it to the description 👍